Source/Core/DolphinWX/Main.cpp

```diff
@@ -147,6 +147,14 @@ bool DolphinApp::OnInit()
 		SConfig::GetInstance().m_coutEnabled = true;
 #endif
 
+	if (m_select_output_directory && !m_output_directory.empty())
+        {
+                std::string output_directory = WxStrToStr(m_output_directory);
+                if (output_directory.back() != '/')
```

Comment:
should also check for `\\` cause windows.

Reply:
Done.